myapp/app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  # 渡されたユーザーでログインする
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  # 現在ログイン中のユーザーを返す (いる場合)
+  def current_user
+    return unless session[:user_id]
+
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  # 受け取ったユーザーがログイン中のユーザーと一致すればtrueを返す
+  def current_user?(user)
+    user == current_user
+  end
+
+  # ユーザーがログインしていればtrue、その他ならfalseを返す
+  def logged_in?
+    !current_user.nil?
```

Comment:
`!` による否定は、使わずに無理なく書ける場合は使わずに書いたほうがわかりやすいと思いますー
```suggestion
    current_user.present?
```

Reply:
ありがとうございます、修正します